e3db/sodium_crypto.py

```diff
@@ -190,7 +191,8 @@ def decrypt_file(self, encrypted_filename, destination_filename, key):
         # whether or not it is used. Since 4KB is already going to be read,
         # and we know the E3DB file header information is much less than 4KB,
         # this size of read makes sense to extract the header from the file
-        e3db_header_block = encrypted_file_header_handle.read(4096).split(".")
+        h1 = encrypted_file_header_handle.read(4096)
```

Comment:
Is `h1` used anywhere else? Flake8 (linter) will often complain about unused variables.